android-core/src/main/java/com/mparticle/internal/ConfigManager.java

```diff
@@ -1276,6 +1280,20 @@ public ConsentState getConsentState(long mpid) {
         return ConsentState.withConsentState(serializedConsent).build();
     }
 
+    public boolean podRedirectionEnabled() {
+        return directUrlRouting;
+    }
+
+    public String getPodPrefix() {
+        String prefix = "us1";
+        try {
```

Comment:
Isn't it generally bad practice to use exceptions for control flow? I think it would be better to just call split, then check the length with an if statement rather than relying on the exception to fall back to the us1 case...